src/account.c

```diff
@@ -1150,6 +1183,21 @@ const char *account_call_transfer(const struct account *acc)
 	return acc->refer ? "yes" : "no";
 }
 
+/**
+ * Get call remote control capability of an account
+ *
+ * @param acc User-Agent account
+ *
+ * @return "yes" or "no"
+ */
+const char *account_remote_control(const struct account *acc)
+{
+	if (!acc)
+		return "no";
+
+	return acc->remote_control ? "yes" : "no";
```

Comment:
I think it is better to use "broadsoft" as a value here, then it is possible to add other
profiles in the future